packages/components/psammead-brand/README.md

```diff
@@ -57,6 +62,8 @@ const Header = (product, serviceName) => (
       minWidth={180}
       svg={igbo}
       url="https://www.bbc.co.uk/news"
+      backgroundColour={backgroundColour}
+      logoColour={logoColour}
```

Comment:
LGTM, but what about adding the `color` addon so we can also change the colors from the Storybook?

```
import {  color } from '@storybook/addon-knobs';
const backgroundColour = color('POSTBOX', `${C_POSTBOX}`);
const logoColour = color('WHITE', `${C_WHITE}`);
```

Reply:
Thanks. Have updated it. I used the labels `Background colour` and `Logo colour` to keep them in keeping with the other knob labels that refer to the prop name.